grafonnet/grafana.libsonnet

```diff
@@ -23,5 +23,6 @@
   heatmapPanel:: import 'heatmap_panel.libsonnet',
   dashlist:: import 'dashlist.libsonnet',
   pluginlist:: import 'pluginlist.libsonnet',
-  gauge:: import 'gauge.libsonnet',
+  gaugePanel:: import 'gauge_panel.libsonnet',
```

Comment:
The rename is a bit confusing, could we raise an error for people using the old gauge panel?
```
gauge:: error 'gauge is removed, migrate to gaugePanel instead`,
```

Reply:
Done.